Logger/LogFactory.cs

```diff
@@ -2,10 +2,27 @@
 {
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private string? _FilePath;
+
+        public string? FilePath { get => _FilePath;  }
+
+        public BaseLogger? CreateLogger(string className)
         {
-            
-            return null;
+            if (_FilePath is null)   
+                return null;
+
+            FileLogger fileLogger = new(_FilePath!)
```

Comment:
```suggestion
            FileLogger fileLogger = new(_FilePath)
```
You shouldn't need the `!` operator here.